packages/navigator/src/browser/navigator-widget.tsx

```diff
@@ -153,7 +153,25 @@ export class FileNavigatorWidget extends FileTreeWidget {
         if (this.model.root && this.isEmptyMultiRootWorkspace(model)) {
             return this.renderEmptyMultiRootWorkspace();
         }
-        return super.renderTree(model);
+        if (model.root) {
+            const rows = Array.from(this.rows.values());
+            if (this.props.virtualized === false) {
+                this.onRender.push(Disposable.create(() => this.scrollToSelected()));
+                return rows.map(row => <div key={row.index}>{this.renderNodeRow(row)}</div>);
+            }
+            return <TreeWidget.View
+                ref={view => this.view = (view || undefined)}
+                width={this.node.offsetWidth}
+                height={this.node.offsetHeight}
+                rows={rows}
+                renderNodeRow={this.renderNodeRow}
+                scrollToAlignment={this.model.scrollAlignment}
+                scrollToRow={this.scrollToRow}
+                handleScroll={this.handleScroll}
+            />;
+        }
+        // eslint-disable-next-line no-null/no-null
+        return null;
```

Comment:
Simply use `undefined` instead?

```suggestion
        return undefined;
```